### DISCUSSION TITLE
whatsmeow deploy on ec2 but not working

### DISCUSSION THREAD

**Discussion by rs2152** [tulir/whatsmeow#604] [WhatsApp protocol Q&A]
Hi team,

I'm trying to use whatsmeow for whatsapp communication, when i'm using the local system ( linux: ubuntu), it is working fine,
but when i deploy the go application on EC2 (Ubuntu), QR is generating but when i try to scan it is showing not connected, Please check your internet connection, I have open all the port on EC2.

Can you please help here.

**Comment by masewo** [comment]
It's very likely that the public IP addresses of AWS are blocked by WhatsApp. Did you trying using a proxy or VPN to a domestic IP?

**Reply by rs2152** [reply]
Hi @masewo , I'm not using the any proxy or VPN.
I just created the EC2 instance and open the all the port. and deploy the go application.
There is any document available where I can follow the steps?
And how I can check my ec2 instance IP is blocked?
